tc-package.sh

```diff
@@ -8,23 +8,25 @@ mkdir -p ${TASKCLUSTER_ARTIFACTS} || true
 
 cp ${DS_ROOT_TASK}/DeepSpeech/tf/bazel_*.log ${TASKCLUSTER_ARTIFACTS}
 
+OUTPUT_ROOT="${DS_ROOT_TASK}/DeepSpeech/tf/bazel-bin"
+
 for output_bin in                                                            \
-    tensorflow/libtensorflow_cc.so                                           \
-    tensorflow/lite/experimental/c/libtensorflowlite_c.so            \
+    "tensorflow/${LIBTENSORFLOW_CC_BASENAME}"                                \
```

Comment:
I'm wondering if it is worth keeping it, since in fact we don't use `libtensorflow_cc.so` at all anymore with static linking. if we decide to continue publishing it, the changes are fine by me.

Reply:
I'm OK with removing this, I was just trying to get the existing system to work.